src/styles/index.css

```diff
@@ -0,0 +1,240 @@
+}
+
+.cards-display {
+  padding: 50px 40px;
+  display: grid;
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (max-width: 487px) {
+  .cards-display {
+    grid-template: auto/200px;
+  }
+}
+
+@media (min-width: 488px) {
+  .cards-display {
+    grid-template: auto/repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards-display {
+    grid-template: auto/repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards-display {
+    grid-template: auto/repeat(4, 200px);
+  }
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  padding: 16px;
+  font-family: Roboto, sans-serif;
+  transition-timing-function: linear;
+  transition-duration: 300ms;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin-top: 16px;
+}
+
+.card__item-model {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgb(6, 11, 53);
+  margin-top: 40px;
+}
+
+.card__item-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: rgb(97, 96, 112);
+  width: 100%;
+  margin-top: 4px;
+}
+
+.card__item-review {
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: flex-end;
+  margin-top: 16px;
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: rgb(6, 11, 53);
+  white-space: nowrap;
+}
+
+.card__item-price {
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__item-price-word {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgb(97, 96, 112);
+}
+
+.card__item-price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 16px;
+  color: rgb(6, 11, 53);
+}
+
+.card__button {
+  margin-top: 16px;
+  height: 40px;
+  width: 100%;
+  border-radius: 5px;
+  background-color: rgb(0, 172, 220);
+  border: 1px solid rgb(0, 172, 220);
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: white;
+  transition-timing-function: linear;
+  transition-duration: 300ms;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+}
+
+.card:hover {
+  transform: scale(120%);
+}
+
+.card:hover .card__item-model {
+  color: #34568b;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-last-child(n + 4),
+.stars--3 .stars__star:nth-last-child(n + 3),
+.stars--4 .stars__star:nth-last-child(n + 2),
+.stars--5 .stars__star:nth-last-child(n + 1) {
+  background-image: url('../images/star-active.svg');
```

Comment:
Ensure that the path to the `star-active.svg` image is correct and that the image is accessible. Incorrect paths can lead to missing images in the UI.